### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by jackshinshin**

### CODE REVIEW

**Comment by ZoneTwelve** [comment]
Can you please give me a description of your PR?

File: app/src/components/Navigation.svelte [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-            <h3 class="text-[0.625rem]">搜尋</h3>
-        </li>
-        <li class="h-10 my-2">
-            <User size='24' />
-        <h3 class="text-[0.625rem]">我的</h3>
-        </li>
-    </ul>
-</nav>
\ No newline at end of file
+	<ul class="w-full fixed inset-x-0 bottom-0 flex justify-around bg-white">
```
Comment (jackshinshin) [inline app/src/components/Navigation.svelte:14]: The following change in this file enables the navigation button to navigate functionally.

**Comment by jackshinshin** [comment]
I've added detailed description to the main change of the files in the PR.

**Comment by ZoneTwelve** [comment]
I have verified with @jackshinshin that this code is obsolete.
I'm going to close this PR.

**Pull request closed without merging**